controllers/runnerpool_controller_test.go

```diff
@@ -478,6 +487,15 @@ var _ = Describe("RunnerPool reconciler", func() {
 					"Name": Equal(secretName),
 				}),
 			}),
+			"NodeSelector": MatchAllKeys(Keys{
```

Comment:
Nice test!
Add to this, could you confirm that the NodeSelector and the Tolerations are not added in the minimal case?

Reply:
@masa213f Thank you for the review.
I have added a test.
Please review again.